src/site/index.rst

```diff
@@ -133,8 +133,11 @@ Labs
     lab2
     lab3
     lab4
-    lab5
 
```

Comment:
Remove empty line?

Reply:
I've added it to have groups of 4 to make parsing the list with my eyes easier.

Reply:
It says `lab` followed by a number. I reject your preference, and substitute my own.